test/dest.js

```diff
@@ -945,6 +945,7 @@ describe('.dest()', function() {
     pipe([
       from.obj([file]),
       destStream,
+      slowCount(1), // Verify and avoid race condition w/ assert check
```

Comment:
I seemed to start having failing assets after upgrading `readable-stream` to v3, including the packages that had v2 as a sub dependency. It feels like a race condition with these tests that I left code comments on. Does this test fix make sense?